src/Pim/Bundle/DataGridBundle/Extension/Formatter/Property/ProductValue/FieldProperty.php

```diff
@@ -32,14 +32,6 @@ protected function convertValue($value)
      */
     protected function getBackendData($value)
```

Comment:
maybe, will be done if needed in an other PR. Here, I just want to show the grid, not refactor it :)